EXILED/Exiled.Loader/Exiled.Loader.csproj

```diff
@@ -17,6 +17,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <PackageReference Include="Mono.Posix-4.5" Version="4.5.0" />
```

Comment:
I think this should not be added

Reply:
Why not?

It is used for the Loader to use Linux, it seemed strange to me that it didn't let me compile and it was because it was missing that dependency.